documentation/chatmeleon-swagger.yaml

```diff
@@ -22,22 +22,14 @@ servers:
 tags:
   - name: conversations
   - name: messages
-  - name: users
 paths:
-  /users/{userId}/conversations:
+  /users/conversations:
     get:
-      description: Retrieve a page of conversations associated with a user, identified by their user ID with the support of cursor-based pagination.
+      description: Retrieve a page of conversations associated with a user, with the support of cursor-based pagination.
```

Comment:
> associated with a user

would `associated with the current logged in user` or something be better?

Reply:
yes ok